config/config.spc

```diff
@@ -1,10 +1,9 @@
 connection "config" {
   plugin = "config"
   
-  # Paths is a list of locations to search for files. Each file will be
-  # converted to a table. Wildcards are supported per
-  # https://golang.org/pkg/path/filepath/#Match
+  # Paths is a list of locations to search for files.
+  # Wildcard based searches are supported.
   # Exact file paths can have any name. Wildcard based matches must have an
```

Comment:
@cbruno10 extensions are not mandatory for wildcard based search, will update the comments.

Reply:
@Subhajit97 We can address updating this comment in https://github.com/turbot/steampipe-plugin-config/pull/8, will help to avoid merge conflicts here (and potentially other places)